operator/images/access-setup/content/bin/setup_work_dir.sh

```diff
@@ -172,6 +172,14 @@ tekton_chains_manifest(){
 }
 
 tekton_results_manifest(){
+  miniosecret="$(kubectl get secrets -n tekton-results | grep "minio-storage-configuration" || true )"
+  if [ -z "$miniosecret" ]; then
+    printf "Need to create tekton results manifests for DB and S3"
```

Comment:
I would prefer removing the config from the kustomization.yaml so there's no need for `install.sh` to know anything about the state of the cluster.

Reply:
I agree conceptually but am uncertain of the implementation.

Would it be `rm -rf $manifests_dir/compute/tekton-results` in `tekton_results_manifest()` when I see that the secret exists ?

Reply:
also, how is it different from what `install.sh` already does in inspecting the chains secret

Reply:
I would use `yq` to remove unnecessary resources from the kustomization.yaml.

We might be doing it wrong for chains too, but don't worry about it now, it's not relevant to that PR.

Reply:
OK looking at the existing yq usage in that method:
`yq e -n '.resources += ["namespace.yaml", "tekton-results-db-secret.yaml", "tekton-results-s3-secret.yaml", "tekton-results-minio-config.yaml"]' > "$results_kustomize"`

I think the simplest thing would be to just (re)create the kustomization.yaml with just the namespace.yaml, which I'm guessing is either:
`yq e -n '.resources += ["namespace.yaml"]' > "$results_kustomize"`

or switch `+=` to `=`:
`yq e -n '.resources = ["namespace.yaml"]' > "$results_kustomize"`

before trying it, I'm guessing the latter, but I should be able to experiment locally

Reply:
either works actually

Reply:
ok took a couple if iterations but was able to test successfully locally with `dev_setup.sh --use-current-branch`

let's see how it does in CI @Roming22